modules/core/src/main/java/org/locationtech/jts/geom/Coordinate.java

```diff
@@ -400,6 +400,15 @@ public Object clone() {
   public Coordinate copy() {
     return new Coordinate(this);
   }
+  
+  /**
+   * Create a new Coordinate of the same type of this Coordinate, but with no values.
+   * 
+   * @return a new Coordinate
+   */
+  public Coordinate createEmptyCopy() {
```

Comment:
Would prefer this called `create`, for brevity

Reply:
Fixed